### PR TITLE
Merge Stripe-java v27.0.0 to beta branch

### PR DESCRIPTION
**Pull request by prathmesh-stripe**
Used the merge.sh script to merge changes from master branch to beta branch. 

## Changelog
* Add support for `reporting_chart` on `AccountSessionCreateParams.components`
* Remove support for `from_schedule` on `Quote.subscription_data`
* Add support for `allow_redisplay` on `terminal.ReaderCollectPaymentMethodParams.collect_config`

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,5 +18,9 @@
   },
   "java.configuration.updateBuildConfiguration": "automatic",
   // LSP was ooming and it recommended this change
-  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -Xlog:disable"
+  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -Xlog:disable",
+  "java.test.config": {
+    "vmargs": [ "-Dstripe.disallowGlobalResponseGetterFallback=true"]
```
Comment (ramya-stripe) [inline .vscode/settings.json:23]: This got pulled in via https://github.com/stripe/stripe-java/pull/1880 - do we know why we needed this?

File: src/main/java/com/stripe/net/ApiRequestParams.java [hunk 2]
```diff
@@ -11,7 +11,7 @@ public abstract class ApiRequestParams {
    * Param key for an `extraParams` map. Any param/sub-param specifying a field intended to support
    * extra params from users should have the annotation
    * {@code @SerializedName(ApiRequestParams.EXTRA_PARAMS_KEY)}. Logic to handle this is in {@link
-   * ApiRequestParamsConverter}.
+   * ApiRequestParamsConverter}.t
```
Comment (ramya-stripe) [inline src/main/java/com/stripe/net/ApiRequestParams.java:14]: What is this weird `.t`? :)

File: src/main/java/com/stripe/net/BaseAddress.java [hunk 3]
```diff
@@ -7,5 +7,7 @@ public enum BaseAddress {
   /** https://connect.stripe.com */
   CONNECT,
   /** https://files.stripe.com */
-  FILES
+  FILES,
+  /** https://events.stripe.com */
```
Comment (ramya-stripe) [inline src/main/java/com/stripe/net/BaseAddress.java:11]: This should be meter-events

**Review by ramya-stripe — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (jar-stripe) [reply .vscode/settings.json:23]: this is needed to get one of our unit tests to run in vs code.  see https://github.com/stripe/stripe-java/pull/1875

**Pull request merged**